tensorflow/core/grappler/utils/topological_sort.cc

```diff
@@ -96,8 +96,8 @@ Status ComputeTopologicalOrder(
                  "at node = "
               << graph.node(back).DebugString();
       for (int i = 0; i < graph_view.num_nodes(); ++i) {
-        if (num_ready_inputs[i] !=
-            static_cast<int>(graph_view.GetFanin(i).size())) {
+        const int graph_view_GetFanin_i_size = graph_view.GetFanin(i).size();
```

Comment:
Variable name should be `snake_case`, not `snake_case_WithCamelCase_and_snake_case_again`